lld/ELF/Arch/AArch64.cpp

```diff
@@ -398,6 +400,24 @@ void AArch64::relocate(uint8_t *loc, const Relocation &rel,
   case R_AARCH64_PREL64:
     write64(loc, val);
     break;
+  case R_AARCH64_AUTH_ABS64:
+    // - If val is wider than 32 bits, we must have converted the auth relr
```

Comment:
Avoid inventing mixing terms "auth relr", ".relr.auth.relr" everywhere. Just stick with the section name  .relr.auth.dyn , which helps code search.

If val is wider than 32bits, the relocation must have been moved from .relr.auth.dyn to .rela.dyn, and the addend write is not needed.

Reply:
Fixed, thanks, see d32b8e32fc1c938e5a005997b832d215a308037d